.github/CONTRIBUTING.md

```diff
@@ -118,45 +119,83 @@ This project uses [Prettier](http://prettier.io/) for code formatting. Consider
+      - **test**: A test-related change.
+  - A subject:
+    - Must be capitalized;
+    - Must be limited to 50 characters or less;
+    - Must omit any trailing punctuation;
+- Header, body and footer must be separated by a blank line;
+- The following lines should have 72 characters or less;
+- Reference to issues and pull requests must be made in the last line.
+
+You also should follow these general guidelines when committing:
+
 - Use the present tense ("Add feature" not "Added feature");
 - Use the imperative mood ("Move cursor to..." not "Moves cursor to...");
-- The first line should:
-  - Be capitalized;
-  - Have 50 characters or less;
-  - Omit any trailing punctuation;
-  - Be followed by a blank line.
-- The following lines should have 72 characters or less;
-- Reference issues and pull requests liberally after the first line;
-- When creating a commit try to answer the following questions:
+- Try to answer the following questions:
   - Why is this change necessary?
   - How does it address the issue?
   - What side effects (if any) does this change may have?
 
-Example:
+Example of a commit message:
 
 ```
-Commit message style guide for Git
+type: Commit message style guide for Git
 
 The first line of a commit message serves as a summary.  When displayed
 on the web, it's often styled as a heading, and in emails, it's
-typically used as the subject. As such, you should capitalize it and
+typically used as the subject. As such, you should specify a "type" and
+a "subject". The type must be lowercase and one of: chore, ci, docs,
+feat, fix, test. For the subject you'll need capitalize it and
 omit any trailing punctuation. Aim for about 50 characters, give or
 take, otherwise it may be painfully truncated in some contexts. Write
-it, along with the rest of your message, in the present tense and imperative mood: "Fix
-bug" and not "Fixed bug" or "Fixes bug". Consistent wording makes it
-easier to mentally process a list of commits.
+it, along with the rest of your message, in the present tense and
+imperative mood: "Fix bug" and not "Fixed bug" or "Fixes bug".
+Consistent wording makes it easier to mentally process a list of
+commits.
 
-Oftentimes a subject by itself is sufficient.  When it's not, add a
+Oftentimes a subject by itself is sufficient. When it's not, add a
 blank line (this is important) followed by one or more paragraphs hard
-wrapped to 72 characters.  Git is strongly opinionated that the author
+wrapped to 72 characters. Git is strongly opinionated that the author
 is responsible for line breaks; if you omit them, command line tooling
 will show it as one extremely long unwrapped line. Fortunately, most
 text editors are capable of automating this.
 
-When relevant, issues and pull request can be referenced using theirs numbers: #3 #12
+Issues and pull request can be referenced on the footer: #3 #12
 ```
 
+### Why all this rules?
+
+We try to enforce that rules for the following reasons:
```

Comment:
`that` -> `these`.

Reply:
Nice! One question, is the `this` in the sub-tittle is correct or would be better if I changed to `these` too?

Reply:
Hmmm.. You got me. Part of me is saying that it would be better if we changed it to `these`. But it is based on a inner grammar-sense. So I could be wrong though.